tools/ci.psm1

```diff
@@ -143,8 +143,8 @@ Function Set-BuildVariable
 
     if($env:TF_BUILD)
     {
-        #In VSTS
-        Write-Host "##vso[task.setvariable variable=$Name;]$Value"
+        # In VSTS
+        Write-Verbose "##vso[task.setvariable variable=$Name;]$Value"
```

Comment:
the ##vso messages must be `write-host` . azure dev ops parses them from the host and looks for `##vso` at the start of the line.

Reply:
Oh ok